application/ledger_query_service/src/queries/ethereum.rs

```diff
@@ -73,22 +73,25 @@ impl ExpandResult for EthereumTransactionQuery {
             .collect();
 
         stream::futures_ordered(futures)
-            .filter_map(|item| item)
+            .filter_map(|item| item) // todo remove me?
```

Comment:
:question: Why is this needed? :open_mouth:  Also the TODO should probably be fleshed out :stuck_out_tongue:

Reply:
Good catch... it's actually needed. I've removed the TODO